CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Breaking changes
 
 - Changed default embeddings model from `text-embedding-ada-002` to `text-embedding-3-small`
+- Added dotnet-isolated support for Embeddings
```

Comment:
This needs to be moved into a `### Changes` or `### Added` section instead of `### Breaking changes`